portfolio-project-matching-app/components/Nav.js

```diff
@@ -1,19 +1,42 @@
+import { useRouter } from 'next/router'
 import Link from 'next/link'
+import Button from './Button'
+import styles from './Nav.module.css'
+import { useEffect, useState } from 'react'
 // import tailwind from 'tailwind/tailwindcss.css'
 
 const Nav = () => {
+    let router = useRouter();
+    const [path, setPath] = useState('');
+    useEffect(
+        () => {
+            setPath(router.pathname);
+        }
+    )
     return (
-        <nav className="mySillyStyle">
-            <ul className="flex">
-                <li className="mr-6">
-                    <Link href='/' >
-                        <a className="text-white text-lg hover:text-blue-100">Home</a>
+        <nav className={styles.nav}>
+            <ul className="grid grid-cols-3">
+                <li className="col-start-2 justify-self-center m-2">
+                    <Link href="/">
+                        <a className="col-start-2 font-medium text-3xl  text-gray-900">Project Finder</a>
                     </Link>
                 </li>
-                <li className="mr-6">
-                    <Link href='/sillyPage' >
-                        <a className="text-white text-lg hover:text-blue-100">sillyPage</a>
-                    </Link>
+                <li className="justify-self-end">
+                    {   path === "/myProfile" ?
+                        <Link href="/" passHref>
+                            {/* Link's child must take an href, so Button must be wrapped with <a> tags */}
+                            <a>
+                            <Button text="Browse" isLink={true} addClassName="bg-white"/>
+                            </a>
+                        </Link>
+                        :
+                        <Link href="/myProfile" passHref>
+                            <a>
+                            <Button text="Profile" isLink={true} addClassName="bg-white"/>
+                            </a>
+                        </Link>
+
+                    }
```

Comment:
Not sure I've had the opportunity to use `useRouter`, thanks for demonstrating it here. I was able to clearly see how you used the path variable to determine what should be displayed.